README.md

```diff
@@ -47,10 +79,40 @@ If you want to launch WRF-CMake and WPS-CMake built in `dmpar` to run on multipl
 - On Linux, use your package manager to download mpich (version ≥ 3.0.4). E.g. `sudo apt-get update && sudo apt-get install mpich` on Debian-based systems or `sudo yum install mpich` on RPM-based system like CentOS.
 
 
-### Source distribution
-To build WRF-CMake from source, see [this page](doc/cmake/INSTALL.md).
+## Documentation
+
+- For the WRF model technical documentation, please refer to [A Description of the Advanced Research WRF Version 3](https://opensky.ucar.edu/islandora/object/technotes%3A500/datastream/PDF/view).
+- For the WRF model user documentation, please refer to [The Advanced Research WRF version 4 Modeling System User’s Guide](http://www2.mmm.ucar.edu/wrf/users/docs/user_guide_v4/contents.html).
+
+## Example usage
+
+If you have already used WRF/WPS before and you just want a quick tutorial to go over the main steps, we have put together a very basic tutorial on our sister-project's website [GIS4WRF](https://gis4wrf.github.io/) with step-by-step instructions [here](https://gis4wrf.github.io/tutorials/wrf-cmake/simulate-the-2018-european-heat-wave-with-wrf-cmake/).
```

Comment:
Please never use "here" as link word. It's too 90's.

Reply:
Full path instead?

Reply:
No, either the title or something descriptive.

Reply:
done in cb63872